WordPress/Classes/ViewRelated/Blog/SiteIconPickerViewController.swift

```diff
@@ -0,0 +1,151 @@
+import Foundation
+import SVProgressHUD
+import WPMediaPicker
+import WordPressComAnalytics
+
+
+/// Encapsulates the interactions required to capture a new site icon image, cropt it and resize it.
+///
+class SiteIconPickerViewController: UIViewController, WPMediaPickerViewControllerDelegate {
```

Comment:
I'll look into that. I was mostly following what we are doing for the user Gravatar but that approach sounds simpler.